docroot/themes/humsci/humsci_basic/humsci_basic.theme

```diff
@@ -274,3 +274,24 @@ function humsci_basic_preprocess_views_view(&$variables) {
       break;
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function humsci_basic_preprocess_paragraph(&$variables) {
+  $paragraph = $variables['paragraph'];
+  if ($paragraph->bundle() == 'hs_view') {
```

Comment:
so many nested conditions. use escape conditions to make things more easily readable.

Reply:
Fixed.